### ISSUE TITLE
Pancake Swap unable to access home page

### ISSUE THREAD

**Issue by azdx1812** [MetaMask/eth-phishing-detect#4845]
I do not know what is going on.....There is not any security concern on my part about pancakeswap.

Please unblock my access.

Best.

**Comment by 409H** [comment]
Closing. Duplicate of https://github.com/MetaMask/eth-phishing-detect/issues/4803
